### PR TITLE
(Part 3 of 3) [breseq Parser] Add initial parse command

### PR DESCRIPTION
**Pull request by phoenixcoder**
I normally don't like/recommend stacking branches on top of one another with different changes, but I thought it'd be good for you @kimlisa and @wai-swk to see how the breseq changes are meant to be used. This relies on this PR getting pushed through: https://github.com/bio-pdv/tools/pull/1

On the command-line a user can do the following for parsing:
```
# This will default to a csv output on the terminal with no logging.
gene parse -f <filepath>

# This will default to a csv output on the terminal with the tool reporting progress status.
gene parse -f <filepath> --status 

# This will output the contents of the file with tsv output on the terminal with the tool reporting progress status.
gene parse -f <filepath> --status --ot tsv
```

Here's what the help output looks like:
```
gene help
Gene is a devops data tool set for performing CRUD operations on the bio-pdv service's database.

Usage:
  gene [flags]
  gene [command]

Available Commands:
  delete      Deletes sequence annotation records from the database.
  help        Help about any command
  parse       Parses a sequence annotation file.
  search      Search for sequence annotation records in the database or file.
  update      Updates sequence annotation records in the database.
  upload      Uploads sequence annotation file(s) to the database.

Flags:
  -d, --debug    Turns on debug logging.
  -h, --help     help for gene
      --status   Turns on reporting of tool progress.

Use "gene [command] --help" for more information about a command.
```

### CODE REVIEW

**Comment by phoenixcoder** [comment]
I normally unit test only the functions the command delegates to. So, in this case, I'd only unit test maybe the validation and/or the printing.

I start to split them up as I'm adding all the other commands to this tool.

File: gene/cmd/root.go [hunk 1]
```diff
@@ -60,22 +95,51 @@ var rootCmd = &cobra.Command{
...
-			log.Fatal(err)
+			fmt.Printf("Could not parse the file. Error: '%s'\n", err.Error())
 		}
 
-		// TODO Validate application name and version.
-		log.Printf("Parsing File: %s\n", filePath)
-		// TODO Add the following back after parser is PR'd
-		//parse.MustParseSeqAnnotationDataFilePath(filePath, "html", "breseq", "0.27")
+		delim := csvDelimiter
+		outputType, err := cmd.Flags().GetString(outputTypeFlag)
```
Comment (kimlisa) [inline gene/cmd/root.go]: I am not seeing this `err` being used.

Comment (phoenixcoder) [reply gene/cmd/root.go]: @kimlisa yeah, I don't use it. If the output type doesn't equal tsv, then I just assume it's csv. I've replaced err with a "_" to indicate that now.

**Comment by phoenixcoder** [comment]
Pushed from desktop.

**Pull request closed without merging**